### ISSUE TITLE
Display images returned by API

### ISSUE THREAD

**Issue by meztez** [mrin9/RapiDoc#251]
Could it be possible to display image returned by API instead of the raw bytes. One issue I think you could have is breaking the layout so it would probably need to be iframed or something.

I saw the PR but it only covers png. A more generalized approache would be prefered.

Let me know if it is something you are interested in, I can work on a PR.

Great customizations by the way.

Example response:
```
�PNG
`...
```

```
content-length: 3758
content-type: image/png
date: Sun, 28 Jun 2020 15:06:41 GMT
```

**Comment by meztez** [comment]
Just saw the PR. Closing.

**Comment by mrin9** [comment]
I will take a PR on this. 
Currently, we do allow to receive binary content type, in that case it shows a download button. However this would work when  the response `content-type = application/octet-stream`

So try changing your response `content-type` to `application/octet-stream` and test it, if it works as you would expect then, you need to add additional condition at the below line to support  some of the popular binary responses `images` , `video`, `audio`, `zip` etc
https://github.com/mrin9/RapiDoc/blob/master/src/components/api-request.js#L1006

**Comment by meztez** [comment]
Closed by #252
